src/main/java/tictactoe/GamePlay.java

```diff
@@ -66,7 +66,7 @@ private int getMoveBetween(int min, int max) {
     }
 
     private boolean isFree(int move) {
-        return this.board.freePosition().contains(move);
+        return this.board.freePositions().contains(move);
```

Comment:
You could add `board.isVacantAt(move)` method here to shift some of the logic down to `Board`